src/Compilers/Core/Portable/Serialization/ObjectReader.cs

```diff
@@ -32,16 +30,18 @@ internal sealed partial class ObjectReader : IDisposable
         /// this version, just change VersionByte2.
         /// </summary>
         internal const byte VersionByte1 = 0b10101010;
-        internal const byte VersionByte2 = 0b00000111;
+        internal const byte VersionByte2 = 0b00001000;
```

Comment:
Changed hte serialization formt for how we write out IObjectWritables.  Previously it was "ObjectTag-TypeTag-TypeId-...data...".  Now it is "ObjectTag-TypeId-...data...".